src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/OpprettBehandling/OpprettBehandlingValg.tsx

```diff
@@ -216,6 +246,36 @@ const OpprettBehandlingValg: React.FC<IProps> = ({
                 />
             )}
 
+            {erMigreringFraInfotrygd && erHelmanuellMigrering && valgteBarn?.erSynlig && (
```

Comment:
erMigreringFraInfotrygd er allerede sjekket for i erHelmanuellMigrering, vurder om den kan fjernes.